### PR TITLE
feat: Resume from my tickets after LOGIN

### LINKED ISSUE #497: when redirected from my tickets to login fragment, resume from my tickets after login
working on it 👍

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #497

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt [hunk 1]
```diff
@@ -108,10 +108,13 @@ class LoginFragment : Fragment() {
 
     private fun redirectToMain(bundle: Bundle?) {
         val intent = Intent(activity, MainActivity::class.java)
-        if (((bundle != null) && !id.equals(-1)) && (ticketIdAndQty != null)) {
-            intent.putExtra(LAUNCH_ATTENDEE, true)
+        if (bundle != null) {
+            if ((!id.equals(-1)) && (ticketIdAndQty != null)) {
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: not required, these 2 if statements can be combined just like it was before

**Comment by nikit19** [comment]
@liveHarshit also please add "fixes" issue number in the description so that when your PR gets merged the issue will also get closed

**Comment by iamareebjamal** [comment]
Please follow commit guidelines

**Comment by liveHarshit** [comment]
> @liveHarshit also please add "fixes" issue number in the description so that when your PR gets merged the issue will also get closed

Okay, Got it. :+1:

File: app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt (hunk 1 above)
Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: Remove unneeded paranthesis

Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: > Remove unneeded parenthesis

Done :+1:

Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: > Remove unneeded parenthesis

Done :+1:

File: app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt [hunk 2]
```diff
@@ -108,10 +108,13 @@ class LoginFragment : Fragment() {
...
         val intent = Intent(activity, MainActivity::class.java)
-        if (((bundle != null) && !id.equals(-1)) && (ticketIdAndQty != null)) {
-            intent.putExtra(LAUNCH_ATTENDEE, true)
+        if (bundle != null) {
+            if (!id.equals(-1) && ticketIdAndQty != null) {
+                intent.putExtra(LAUNCH_ATTENDEE, true)
+            }
             intent.putExtras(bundle)
         }
+
```
Comment (simarsingh24) [inline app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: Extra feed

Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: Sorry about that, I will take care of these by next time.

Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: > Extra feed

Sorry about that, I will take care of these by next time.

File: app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt [hunk 3]
```diff
@@ -108,10 +109,13 @@ class LoginFragment : Fragment() {
...
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: Extra

Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: > Extra

removed

**Comment by liveHarshit** [comment]
Updated, Sorry for all the complications. I will try best by next time.

**Pull request merged**